test/features/my_academics.feature

```diff
@@ -0,0 +1,7 @@
+Feature: My Academics Enrollment Page
+  User should be able to click on schedule of classes,add,drop,swap,edit classes
```

Comment:
@davidlimcheng That might be the point of the private [ucberkeley/sis-qa-testcases](https://github.com/ucberkeley/sis-qa-testcases) repo. I agree that the QA team would benefit from having examples that describe a similar feature in two contexts, but they'll probably get that with the examples that @mDibyo creates there.

`Given I login to calcentral with valid faculty credentials`
vs
`Given I login to calcentral with valid student credentials`